### ISSUE TITLE
CPU-miners: run on 'low priority'

### ISSUE THREAD

**Issue by eMTee72** [tutulino/Megaminer#214]
When running a CPU-miner, system can become less responsive to user input (and eventually also the MegaMiner PowerShell script and feeding the GPU mining applications with new data). This is mainly applicable to systems with a lower-end CPU (but also with more powerful CPU's!).
Therefor, it would make sense to start CPU miners with a Low Priority.
This can be achieved in several ways:

1. using the options from the CPU miner itself: as an example XMRig has the option '--cpu-priority=0'

2. generic approach: using the Windows command START /LOW. E.g.: "_start /low xmrig.exe -o stratum+tcp://europe.cryptonight-hub.miningpoolhub.com:20580 -u My.User_CPU:x --api-port 2520 --donate-level=1 --max-cpu-usage=80 --print-time=60_"

This may have a minor impact towards the performance of the CPU miner, but should benefit the GPU miners.

**Comment by Nexworks** [comment]
You are correct, all the manual CPU miners I run are Start /low  and its absolutely worth it.
